app/src/ui/history/commit-summary.tsx

```diff
@@ -195,37 +199,50 @@ export class CommitSummary extends React.Component<
     )
   }
 
-  private renderExternalLink() {
-    if (this.props.isLocal) {
-      return null
-    }
-
+  private onShowCommitOptions = () => {
+    let label: string = ''
     const gitHubRepository = this.props.repository.gitHubRepository
-    if (!gitHubRepository) {
-      return null
-    }
 
-    const url = `${gitHubRepository.htmlURL}/commit/${this.props.sha}`
-    const isDotCom = gitHubRepository.endpoint === getDotComAPIEndpoint()
+    if (gitHubRepository !== null) {
```

Comment:
Generally we should just say `if (gitHubRepository)` vs. comparing directly to `null` or `undefined`, because we don't want to fall into the branch if it's `undefined`.